scripts/gufi_stat

```diff
@@ -133,6 +139,9 @@ def run(argv, config_path):
     paths = [os.path.normpath(os.path.sep.join([config.indexroot(), path]))
              for path in args.paths]
 
+    if args.verbose:
```

Comment:
@calccrypto , here is where the verbose print is happening in gufi_stat. It's before the call to subprocess so I'm confused as to why the verbose is printed after the output. When I run ./test/regression/gufi_stat.sh, it does show it backwards with the output first and then the verbose printout. But when I run the command at the command line, it shows the verbose printout before the output as expected. So, I'll keep looking but this is confusing.

Reply:
It was just buffering. I think I have it fixed.

Reply:
Whoops, my fix was python3 which I learned because the automated testing of py2 was failing. Updated to use a flush mechanism compatible with py2 and py3